app/controllers/api/v1/api_controller.rb

```diff
@@ -8,6 +8,28 @@ class ApiController < ApplicationController
     rescue_from Apipie::ParamInvalid, with: :invalid_params
     rescue_from Apipie::ParamMissing, with: :invalid_params
 
+    def serializer_for(object)
+      # using safe_constantize as it will return nil if no object is found 
```

Comment:
Do not add comment inside method. If you want this comment then add it before method start. Also i don't think we need comment here as its behaviour of `safe_constantize`.